### PR TITLE
Feat/lesson 5 code blocks update

### PR DESCRIPTION
**Pull request by Markkos89**
## Changes

- Code block on the whole lesson updated
- Some text and tools description texts were improvised, open to suggestions or changes. Just add a new PR :))

## API Updates

No new API updates

### New Features *(required)*

Code block on the whole lesson updated

Fixes #
Connects #

### CODE REVIEW

**Comment by elPiablo** [comment]
Muchos gracias @Markkos89 !!!!

**Comment by Markkos89** [comment]
@elPiablo this PR should go to `staging` rather to `main` right? let me know and I will change it, thanks!

File: src/pages/lessons/projects/5.mdx [hunk 1]
```diff
@@ -23,274 +17,322 @@ import Layout from "../../../components/Layout";
...
+them. Once minted, users will be redirected to a page that displays information
+of their mint. We'll need some extra magic to make that all happen. For handling
+wallet connections to our application, we'll be introducing you to _RainbowKit_.
+And using the provided _hooks_ from _WAGMI_, we can check the connection to the
+wallet and call functions from our smart contract through the frontend.
+_RainbowKit_ and _WAGMI_ are just another couple of shining examples of open
+source public goods. We love open source at _Developer_DAO_. And public goods
+are gooood!
+
+- [RainbowKit](https://www.rainbowkit.com/docs/introduction) for out-of-the-box
```
Comment (elPiablo) [inline src/pages/lessons/projects/5.mdx]: delete:
'[RainbowKit](https://www.rainbowkit.com/docs/introduction) for out-of-the-box wallet management (i.e. Handling wallet connections to our application).

[WAGMI](https://wagmi.sh/) for checking connection to a wallet and calling functions from our smart contract through the frontend using the provided hooks. */}'

File: src/pages/lessons/projects/5.mdx [hunk 2]
```diff
@@ -23,274 +17,322 @@ import Layout from "../../../components/Layout";
...
 
 {/* We could do with a oneliner for each to  describe what index.js and app.js are for */}
+
 ## Let's breakdown the lesson
+
 - Set up our development environment with TierNFT project
-- initialise a nextJS app 
+- initialise a nextJS app
 - Install RainbowKit, WAGMI and Ethers to create main application
-  - import dependencies
```
Comment (elPiablo) [inline src/pages/lessons/projects/5.mdx:54]: the original was good (maybe a slip with the enter key?) 
Please replace with 
- import dependencies

**Review by elPiablo — APPROVED** [review]
Gtg

**Pull request merged**